android/src/main/java/com/decodedhealth/flutter_zoom_plugin/ZoomView.java

```diff
@@ -77,6 +82,7 @@ private void init(final MethodCall methodCall, final MethodChannel.Result result
         }
 
         ZoomSDKInitParams initParams = new ZoomSDKInitParams();
+        initParams.jwtToken = options.get("sdkToken");
```

Comment:
Why is the initialisation jwt token is used with sdk token here?

Reply:
You can provide the appKey and appSecret or a jwtToken to the SDK, the jwtToken is more secure because you don't have to hardcode the values of the Key and Secret.

Here is the doc [Zoom Initialization](https://marketplace.zoom.us/docs/sdk/native-sdks/android/mastering-zoom-sdk/sdk-initialization)

Quoting "If the appKey and appSecret is not passed, a jwtToken should be used."

Reply:
Sorry if I was not clear. I think the token name mapping is not correct here. The JWT token is mapped to the sdk token